apps/voting-disputable/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -21,7 +30,7 @@ function DisputableActionStatus({ vote }) {
   //TODO: get agreement and vote real data
   const agreement = getAgreement()
 
-  const { challengeAmount, collateralToken } = vote.disputable.action.collateral
+  const { actionAmount, decimals, symbol } = vote.disputable.action.collateral
```

Comment:
I think this is going to throw as `disputable` is initially `undefined`, I also don't think we need to provide the entire vote object to this component. How about just passing `disputable` to props and then guarding appropriately? e.g.

```javascript
const tokenAmount = useMemo(() => {
    if (disputable) {
      const { actionAmount, decimals, symbol } = disputable.action.collateral

      return formatTokenAmount(actionAmount, decimals, { symbol: symbol })
    }

    return ''
  }, [disputable])

```